packages/components/src/List/List.component.js

```diff
@@ -8,10 +8,52 @@ import DisplayTile from './DisplayTile';
 import Content from './Content';
 import theme from './List.scss';
 
+function getToolbar(toolbar, id, displayMode, list) {
+	const toolbarProps = {
+		...toolbar,
+		id,
+	};
+
+	if (toolbar.display) {
+		toolbarProps.display.mode = displayMode;
+	}
+
+	if (list.itemProps && list.itemProps.isSelected && list.itemProps.onToggleAll) {
+		toolbarProps.selectAllCheckbox = {
+			id,
+			items: list.items,
+			isSelected: list.itemProps.isSelected,
+			onToggleAll: list.itemProps.onToggleAll,
+		};
+	}
+	return (<Toolbar {...toolbarProps} />);
+}
+
+function getDisplayModeComponent(useContent, id, displayMode, list) {
```

Comment:
it looks like a component -> please use unstructure args (as below)
function getDisplayModeComponent({ useContent, id, displayMode, list } )